core-services/prow/02_config/openshift/cluster-kube-controller-manager-operator/_pluginconfig.yaml

```diff
@@ -6,6 +6,18 @@ label:
       - zhouying7780
       - xingxingxia
       label: cherry-pick-approved
+    - allowed_users:
+      - damemi
+      - danwinship
+      - dcbw
+      - deads2k
+      - ingvagabund
+      - knobunc
+      - mfojtik
+      - soltysh
+      - squeed
+      - sttts
```

Comment:
As the team lead primarily  owning kcm-o I will object, as stated before I'd only pick top-level approvers here, so I'd expect this file to match https://github.com/openshift/cluster-kube-controller-manager-operator/blob/master/OWNERS#L7-L13